armory/scenarios/carla_video_tracking.py

```diff
@@ -24,51 +25,67 @@ def load_dataset(self):
             raise ValueError("batch_size must be 1 for evaluation.")
         super().load_dataset(eval_split_default="dev")
 
+    def next(self):
+        x, y = next(self.test_dataset)
+        i = self.i + 1
+        self.i, self.x, self.y = i, x, y
+        self.y_object, self.y_patch_metadata = y
+        self.y_pred, self.y_target, self.x_adv, self.y_pred_adv = None, None, None, None
```

Comment:
Similarly to above:
This could be simplified to:
```
    def next(self):
        super().next()
        self.y_object, self.y_patch_metadata = self.y
```